documentation/docs/examples/point-to-point-measurement.mdx

```diff
@@ -149,10 +146,19 @@ setTimeout(() => {
 }, 6000);
 ```
 
+### Hide and show measurements
+It's possible to toggle between hide & show the measurements in the viewer.
+It can be useful to view the details of an object without measurements blocking the view.
```

Comment:
```suggestion
It may be useful to view the details of an object without measurements blocking the view.
It's possible to toggle measurement visibility on and off in the viewer.
```